src/thunderbots/software/grsim_communication/main.cpp

```diff
@@ -91,6 +93,8 @@ int main(int argc, char** argv)
         // Clear all primitives each tick
         primitives.clear();
 
+        primitives.emplace_back(
+            new MovePrimitive(0, ball.position(), Angle::full(), 0.5));
```

Comment:
I don't think we should need the changes to this file?

Reply:
testing code, removed